drivers.json

```diff
@@ -3740,9 +3740,18 @@
                     "product": "GeForce",
                     "version": "531.26",
                     "variant": "DCH (Hotfix)",
-                    "patch64_url": "win10_x64/531.18/nvencodeapi64.1337",
```

Comment:
Oops missed this in the last PR. I hope you're not manually editing these, there are helper scripts to do all of this :)

Reply:
I did in the last one and missed my mistake.